### ISSUE TITLE
[FEATURE REQ] RuleManager possibility to update rules

### ISSUE THREAD

**Issue by danielmarbach** [Azure/azure-sdk-for-net#31625]
### Library name

Azure.Messaging.ServiceBus

### Please describe the feature.

The new rule manager is very cool! Kudos.

Would it be possible to support updating rules? Or is that prevented because updating rules requires manage claims?

I did a quick attempt to "hack it into the rule manager" but got 

```
System.UnauthorizedAccessException: Unauthorized access. '' claim(s) are required to perform this operation. 
```

**Comment by jsquire** [comment]
I'm not sure that the service supports updating via the AMQP management link.  If I recall, we included all supported functionality in the rule manager.. but @JoshLove-msft would know for sure.

**Comment by JoshLove-msft** [comment]
> I'm not sure that the service supports updating via the AMQP management link. If I recall, we included all supported functionality in the rule manager.. but @JoshLove-msft would know for sure.

Correct, the service doesn't currently support this, but they are aware of this and have an issue filed to consider adding this support.

**Comment by JoshLove-msft** [comment]
Since the service is already tracking this separately, I will close this issue.
